interfaces/Portalicious/src/app/domains/project/project.helper.ts

```diff
@@ -89,6 +140,17 @@ export function projectHasFspWithExportFileIntegration(project?: Project) {
   );
 }
 
+export function projectHasInclusionScore(project?: Project): boolean {
+  if (project?.programRegistrationAttributes) {
+    for (const attribute of project.programRegistrationAttributes) {
+      if (Object.keys(attribute.scoring).length > 0) {
+        return true;
+      }
+    }
+  }
+  return false;
```

Comment:
```suggestion
  return (project?.programRegistrationAttributes ?? []).some(attribute => Object.keys(attribute.scoring).length > 0));
```

This is how I would've written it, but I don't care too much. Mostly curious what you prefer.

Reply:
Don't be scared!